inlang/packages/sherlock/src/commands/extractMessage.ts

```diff
@@ -13,6 +13,16 @@ import {
 } from "@inlang/sdk"
 import { v4 as uuidv4 } from "uuid"
 import { saveProject } from "../main.js"
+import { customAlphabet } from "nanoid"
+
+/**
+ * Generators used to populate the message key field.
+ */
+const generators = {
+	humanId,
+	nanoid: customAlphabet("abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ", 10),
```

Comment:
```suggestion
	nanoid: customAlphabet("abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ", 8),
```

52^8 offers 12,000 times more combination than the current human id, I'd say that's more than enough and saves 2 bytes per key